status/metrics.go

```diff
@@ -39,6 +41,23 @@ var ConditionDuration = pmetrics.NewPrometheusHistogram(
 	},
 )
 
+func conditionDurationMetric(objectName string) pmetrics.ObservationMetric {
+	return pmetrics.NewPrometheusHistogram(
+		metrics.Registry,
+		prometheus.HistogramOpts{
+			Namespace: MetricNamespace,
+			Subsystem: fmt.Sprintf("%s_%s", objectName, MetricSubsystem),
+			Name:      "transition_seconds",
+			Help:      "The amount of time a condition was in a given state before transitioning. e.g. Alarm := P99(Updated=False) > 5 minutes",
+		},
+		[]string{
+			MetricLabelGroup,
```

Comment:
You aren't going to be able to register this metric twice so I think that we should drop group from here. Same is true of all of the other metrics that we are generating in this file